src/collection.ts

```diff
@@ -340,12 +340,17 @@ export class Collection<TSchema extends Document = Document> {
    * Update a single document in a collection
    *
```

Comment:
I would add the below comment up here and format it a bit differently to get it to render nicely, you can use markdown syntax (ex. bullets and fixed-width):

```ts
  /**
   * Update a single document in a collection
   *
   * The value of `update` can be either:
   * - UpdateFilter<TSchema> - A document that contains update operator expressions,
   * - Document[] - an aggregation pipeline.
   *
   * @param filter - The filter used to select the document to update
   * @param update - The modifications to apply
   * @param options - Optional settings for the command
   */
```

Reply:
Updated with your suggestion.